### PR TITLE
Link to NC 14 dev manual

### PR DESCRIPTION
**Pull request by rullzer**
Master for #8260 

We probably should use the urlgenerator here but there is no go entry for this I think. So lets fix this now and add a ticket for the tech debt later...

### CODE REVIEW

File: settings/templates/apps.php [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 <?php if($_['appstoreEnabled']): ?>
 	<li>
-		<a class="app-external icon-info" target="_blank" rel="noreferrer noopener" href="https://docs.nextcloud.org/server/12/developer_manual/"><?php p($l->t('Developer documentation'));?> ↗</a>
+		<a class="app-external icon-info" target="_blank" rel="noreferrer noopener" href="https://docs.nextcloud.org/server/14/developer_manual/"><?php p($l->t('Developer documentation'));?> ↗</a>
```
Comment (nickvergessen) [inline settings/templates/apps.php]: Can we generate this with the IURLGenerator instead? then it doesnt need manual adjustment all the time

Comment (rullzer) [reply settings/templates/apps.php]: We probably need to add an entry to the go.php thingy but then probably yes. As stated above that sounds way easier ;)

Comment (rullzer) [reply settings/templates/apps.php]: But yeah let me just do that

**Review by nickvergessen — APPROVED** [review]

**Review by MorrisJobke — APPROVED** [review]

**Comment by MorrisJobke** [comment]
CI is not happy

**Pull request merged**